helm/superset/values.yaml

```
@@ -166,7 +166,7 @@ configOverrides: {}
   #   AUTH_USER_REGISTRATION_ROLE = "Admin"
   # secret: |
   #   # Generate your own secret key for encryption. Use openssl rand -base64 42 to generate a good key
```

```suggestion
  #   # Generate your own secret key for encryption. Use `openssl rand -base64 42` to generate a good key
```
Could we also change this to make it easier to spot the command?